docs/source/guides/deployment.ipynb

```diff
@@ -0,0 +1,208 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "92a0dab5",
+   "metadata": {},
+   "source": [
+    "# Deployment\n",
+    "\n",
+    "Deployment of machine learning models requires repeating feature engineering steps on new data. In some cases, these steps need to be performed in near real-time. Featuretools has capabilities to ease the deployment of feature engineering.\n",
+    "\n",
+    "## Saving Features\n",
+    "\n",
+    "First, let's build some generate some training and test data in the same format. We use a random seed to generate different data for the test."
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "id": "129c8011",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note ::\n",
+    "\n",
+    "    Features saved in one version of Featuretools are not guaranteed to load in another. This means the features might need to be re-created after upgrading Featuretools."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "01c19e97",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import featuretools as ft\n",
+    "\n",
+    "es_train = ft.demo.load_mock_customer(return_entityset=True)\n",
+    "es_test = ft.demo.load_mock_customer(return_entityset=True, random_seed=33)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "042f8c02",
+   "metadata": {},
+   "source": [
+    "Now let's build some features definitions using DFS. Because we have categorical features, we also encode them with one hot encoding based on the values in the training data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6bcc87a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_matrix, feature_defs = ft.dfs(entityset=es_train,\n",
+    "                                      target_dataframe_name=\"customers\")\n",
+    "\n",
+    "feature_matrix_enc, features_enc = ft.encode_features(feature_matrix, feature_defs)\n",
+    "feature_matrix_enc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03ffe00a",
+   "metadata": {},
+   "source": [
+    "Now, we can use `featuretools.save_features` to save a list features to a json file"
```

Comment:
Add link to autogenerated API doc for `featuretools.save_features`?

Reply:
Added link - hopefully correctly.